drivers/include/tmp006.h

```diff
@@ -101,6 +101,26 @@ extern "C"
 #endif
 
 /**
+ * @brief   Default low power mode
+ * If set to 0, the device will be always-on
```

Comment:
Add an empty line here to let doxygen know where the brief documentation ends. You can check what it looks like using `make doc` from the RIOT top level directory, then open RIOT/doc/doxygen/html/index.html